packages/conf-libcorosync/conf-libcorosync.3/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+maintainer: "corosync"
+authors: ["Jan Friesse"]
+homepage: "https://corosync.github.io/corosync/"
+license: "BSD"
+build: ["pkg-config" "--print-errors" "--exists" "libcmap" "libcpg" "libcfg" "libquorum" "libvotequorum"]
+depexts: [
+  ["corosynclib-devel"] {os-distribution = "centos"}
+  ["libcfg-dev"] {os-family = "debian"}
+  ["libcmap-dev"] {os-family = "debian"}
+  ["libcpg-dev"] {os-family = "debian"}
+  ["libquorum-dev"] {os-family = "debian"}
+  ["libvotequorum-dev"] {os-family = "debian"}
+  ["libcfg-dev"] {os-family = "ubuntu"}
+  ["libcmap-dev"] {os-family = "ubuntu"}
+  ["libcpg-dev"] {os-family = "ubuntu"}
+  ["libquorum-dev"] {os-family = "ubuntu"}
+  ["libvotequorum=dev"] {os-family = "ubuntu"}
+  ["corosynclib-devel"] {os-distribution = "fedora"}
+  ["lib64corosync-devel"] {os-distribution = "mageia"}
+  ["libcorosync-devel"] {os-family = "opensuse" | os-family = "suse"}
+]
+available: os = "linux" & arch != "arm32" & os-distribution != "alpine" & os-distribution != "arch"
```

Comment:
```suggestion
available: os = "linux" & arch != "arm32"
```

is corosync not installable on alpine and arch? If it can be manually installed, I would rather drop this unavailability condition and let it fail in the CI

Reply:
I think at least it is not available in the package manager of these two distributions. But I presume one can always manually compile and install them there

Reply:
but yeah, I think in theory one could still install corosync manually and use this binding there. I will remove this limitation.